vic/drivers/cesm/src/cesm_interface_c.c

```diff
@@ -34,7 +34,7 @@ all_vars_struct    *all_vars = NULL;
 atmos_data_struct  *atmos = NULL;
 x2l_data_struct    *x2l_vic = NULL;
 l2x_data_struct    *l2x_vic = NULL;
-dmy_struct          dmy;
+dmy_struct          dmy_current;
```

Comment:
The way I've implemented the time keeping in the CESM driver only requires us to have a single `dmy_struct`. We've defined it as a global here and it is passed around as a pointer in the rest of the code.